docs/viplab3.0/computation_template.md

```diff
@@ -243,23 +275,28 @@ Four access levels can be specified inside a part:
 
 ### JSON object Parameter
 
-An PARAM_ID-object, like *\_\_BINARY\_\_*, has the following members:
+A parameter-object, has the following members:
 
 |Key [--Subkey] | Type (a default is marked by _italics_)|Opt / Must |Description |Comment | 
 |---------------|----------------------------------------|-----------|------------|--------|
-|guiType | one of {*"editor"*, "input_field", "checkbox", "radio", "dropdown", "toggle", "slider"} | must | specifies how the frontend renders the parameter | |
-|name | string | must | Label for the parameter | frontend feature |
-|type | one of {"number", "text"} | opt | Type of the input field |  | 
+|type | one of {"any-type", "fixed-type"} | must | specifies type of the parameter | depends on the guiType - *fixed-type*: checkbox, radio, dropdown, toggle; *any-type*: slider, input_field, editor |
+|identifier | PARAM_ID | must | specifies ID of the parameter | Example: "*\_\_BINARY\_\_*" |
+| metadata | Object | must | Object that contains general info about the parameter | |
+| metadata --guiType | one of {*"editor"*, "input_field", "checkbox", "radio", "dropdown", "toggle", "slider"} | must | specifies how the frontend renders the parameter | |
+| metadata --type | one of {"number", "text"} | opt | Type of the input field |  | 
+| metadata --name | string | must | Label for the parameter | frontend feature |
+| metadata --vertical | bool | opt (*false*) | Specifies for *gui_type* "slider" whether it is rendered horizontal or vertical |
 |value | number for numerical "input_field"; array of numbers for a range slider | opt | the default value(s) shown in frontend | |
-|values | array of strings | must for *gui_type* "checkbox", "radio", "dropdown", "toggle" | specifies the allowed values | |
-|min | number | opt | minimal allowed value | |
-|max | number | opt | maximal allowed value | |
-|step | number | opt | defines together with *min* and *max* attributes a finite set of allowed values | | 
+|values | array of objects | must for *gui_type* "checkbox", "radio", "dropdown", "toggle" | specifies the allowed values | |
+|values --value | string | must | specifies one avaliable value | Example: { "value" : "verbose" } |
+|values --disabled | boolean | opt | Shows disabled options in frontend | Example: { "value" : "Please choose multiple", "disabled" : true } |
+|min | number | opt | minimal allowed value | for slider, or input_field with type number |
+|max | number | opt | maximal allowed value | for slider, or input_field with type number |
+|step | number | opt | defines together with *min* and *max* attributes a finite set of allowed values | for slider, or input_field with type number | 
```

Comment:
Also the values and min/max specs should be split up into an one table for fixed-value parameters and one for any-value parameters